web/backend/src/Server.ts

```diff
@@ -151,52 +183,63 @@ app.post('/api/logout', (req, res) => {
   });
 });
 
+// This function helps us convert the double list of the authorization
+// returned by the casbin function getFilteredPolicy to a map that link
+// an object to the action authorized
+// list[0] contains the policies so list[i][0] is the sciper
+// list[i][1] is the object and list[i][2] is the action
+function setMapAuthorization(list: string[][]) {
+  const m = new Map<String, Array<String>>();
+  for (let i = 0; i < list.length; i += 1) {
+    if (m.has(list[i][1])) {
```

Comment:
The comment is already a great improvement. 👍 

Now, the next step is self-explanatory code.. ;) 

You could do:

```
const [subject, action] = list[i][1], list[i][2];
if (m.has(subject)) {
...
```

Reply:
Thanks for the review ! Done